doc/getting_started/index.rst

```diff
@@ -358,17 +369,45 @@ each with their own way to install development tools:
              gnu-efi-devel \
              libuuid-devel \
              openssl-devel \
-             libpciaccess-devel
+             libpciaccess-devel \
+             systemd-devel \
+             libxml2-devel \
+             libevent-devel
 
-Build the hypervisor and device model
-=====================================
 
-#. Download the ACRN hypervisor and build it.
+Build the hypervisor, device model and tools
+============================================
+
+The `acrn-hypervisor <https://github.com/projectacrn/acrn-hypervisor/>`_
+repository has three main components in it:
+
+1. The ACRN hypervisor code located in the ``hypervisor`` directory
+#. The ACRN devicemodel code located in the ``devicemodel`` directory
+#. The ACRN tools source code located in the ``tools`` directory
+
+You can build all these components in one go as follows:
+.. code-block:: console
+
+   $ git clone https://github.com/projectacrn/acrn-hypervisor
+   $ cd acrn-hypervisor
+   $ make
```

Comment:
These instructions are different than those in the UP2 GSG?

Reply:
Not really. This instruction will build everything (well, except the documentation). That means the hypervisor, devicemodel and tools. We only build the hypervisor and devicemodel in the UP2 GSG.